src/styles/block/watch.scss

```diff
@@ -0,0 +1,65 @@
+.watch {
+  padding: 20px;
+
+  &__hand-second {
```

Comment:
Your seconds and minutes hands have a lot of common props. Move them to a common element f.e. `.watch__hand`. And create two modifiers for secs and mins.